governance/votes/Executive Vote - November 15, 2023.md

```diff
@@ -0,0 +1,237 @@
+
+- Increase the Spark Lend Gnosis Chain wstETH [supply cap](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps) from 5,000 wstETH to **10,000 wstETH**.
+- Reduce the Spark Lend Ethereum Dai market [Loan-to-Value (LTV)](https://docs.aave.com/risk/asset-risk/risk-parameters#loan-to-value) from 1bps to **0**.
+- Reactivate the WBTC market on Spark Lend Ethereum with the following parameters:
+	- Maximum [Loan-to-Value LTV](https://docs.aave.com/risk/asset-risk/risk-parameters#loan-to-value): **70%**
+	- [Liquidation Threshold](https://docs.aave.com/risk/asset-risk/risk-parameters#liquidation-threshold): **75%**
+	- Liquidation penalty: **7%**
+	- [Supply Cap](https://docs.sparkprotocol.io/developers/features/supply-borrow-caps#supply-caps): **3,000 WBTC**
+	- [Borrow Cap](https://docs.sparkprotocol.io/developers/features/supply-borrow-caps#borrow-caps): **2,000 WBTC**
+	- [Reserve Factor](https://docs.aave.com/risk/asset-risk/risk-parameters#reserve-factor): **20%**
+	- Base rate: **0%**
+	- Optimal rate: **2%**
+	- Max rate: **302%**
+	- Optimal utilization: **60%**
+	- [Efficiency mode or eMode](https://docs.sparkprotocol.io/developers/features/efficiency-mode-emode): **No**
+	- [Isolation Mode](https://docs.sparkprotocol.io/developers/features/isolation-mode): **No**
+	- Debt ceiling: n/a
+		- ~$78,000,000 effective risk exposure (~10% of TVL, ~155% of Maker surplus)
+- Increase the Spark Lend Ethereum rETH and wstETH [supply caps](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps):
+	- Increase the rETH supply cap from 60,000 rETH to **80,000 rETH**.
+	- Increase the wstETH supply cap from 400,000 wstETH to **800,000 wstETH**.
+- Adjust the Spark Lend Ethereum **and** Gnosis Chain ETH [interest rate model](https://docs.aave.com/risk/liquidity-risk/borrow-interest-rate#interest-rate-model) parameters:
+	- Base rate: **0%**
+	- Optimal Rate: **3.2%**
+	- Optimal Utilization: **90%**
+	- Maximum Rate: **123.2%**
+
+### Adjust Spark Lend D3M Maximum Debt Ceiling (`line`)
+
+*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmVbrypf#poll-detail)\
+*Proposal*: [Forum Post](https://forum.makerdao.com/t/proposal-to-adjust-sparklend-parameters/22542)
+
+If this executive proposal passes, the following change will be made to the Spark Lend Direct Deposit Module Maximum Debt Ceiling:
+
+- Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 400 million DAI to **800 million DAI**.
+
+### Launch Project Funds Transfer
+
+*Authorization*: [Forum Notification](https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/6)
```

Comment:
I would also include this URL: https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget

Reply:
It is included on line 83. We don't think duplication is necessary